node/services/config.go

```diff
@@ -17,32 +17,39 @@ type Config struct {
 	// TrustedHash is the Block/Header hash that Nodes use as starting point for header synchronization.
 	// Only affects the node once on initial sync.
 	TrustedHash string
-	// TrustedPeer is the peer we trust to fetch headers from.
+	// TrustedPeers are the peers we trust to fetch headers from.
 	// Note: The trusted does *not* imply Headers are not verified, but trusted as reliable to fetch headers
 	// at any moment.
-	TrustedPeer string
+	TrustedPeers []string
 }
 
 // TODO(@Wondertan): We need to hardcode trustedHash hash and one bootstrap peer as trusted.
 func DefaultConfig() Config {
 	return Config{
-		TrustedHash: "",
-		TrustedPeer: "",
+		TrustedHash:  "",
+		TrustedPeers: make([]string, 0),
 	}
 }
 
-func (cfg *Config) trustedPeer() (*peer.AddrInfo, error) {
-	if cfg.TrustedPeer == "" {
-		log.Warn("No Trusted Peer provided. Headers won't be synced accordingly")
-		return &peer.AddrInfo{}, nil
+func (cfg *Config) trustedPeers() ([]*peer.AddrInfo, error) {
+	if len(cfg.TrustedPeers) == 0 {
+		cfg.TrustedPeers = params.Bootstrappers()
```

Comment:
We should log this IMO.

```
No trusted peers given, initializing with default bootstrappers as trusted peers
```

Reply:
added log

Reply:
@renaynay, explain why. _Not_ passing trusted peers is going to be a default case after this PR, and there is no point to warn user that their are doing something wrong by using defaults

Reply:
Though Info log is fine, unless it's not a Warning, like it was before

Reply:
No, that's what i meant. It should be an INFO log.

Reply:
thank you for adding the log @vgonkivs , this comment thread is now resolved.